ci/config/autotest.cfg

```diff
@@ -49,3 +49,8 @@ type =
 key =
 server =
 test_project =
+
+[mgmtcenter]
+name =
+username =
+password =
```

Comment:
I think we also need to be able to add a specific type of mgmt center, i.e. esx / kvm, so the config can be used to control what the test should do.